src/datasources/cache/cache.router.ts

```diff
@@ -24,41 +24,46 @@ export class CacheRouter {
   private static readonly TOKENS_KEY = 'tokens';
   private static readonly TRANSFERS_KEY = 'transfers';
 
+  static getBalanceCacheKey(chainId: string, safeAddress: string): string {
+    return `${CacheRouter.BALANCES_KEY}_${chainId}_${safeAddress}`;
```

Comment:
@fmrsabino I tried to maintain a common format for first-level cache keys (`$feature_$chainId`), but this is an exception to that rule. The reason is that according to `cache-hooks.service.ts`, it should be possible to invalidate the balances **by address**, and redis [does not implement sub-key expirations](https://github.com/redis/redis/issues/167), so the options I see here are:
- Use `balances_$chainId` as key and Implement a custom logic to search through the fields using `HSCAN` to retrieve and delete fields matching `$safeAddress_*`.
- Use `balances_$chainId` as key and just delete all the chain balances at once.
- Make an exception and use `balances_$chainId_$address` as key.

I took the last option, but of course is open for discussion. Or if you see more options they will be welcomed 🙂